### ISSUE TITLE
Bug: Table of contents can't be scrolled if too long

### ISSUE THREAD

**Issue by Mindiell** [JohnMcLear/ep_table_of_contents#42]
On this pad : https://annuel2.framapad.org/p/636etyxzoi-9ibk?lang=fr
When you display the table of contents and try to scroll it downside, the table jump to the top automatically.

**Comment by JohnMcLear** [comment]
That's an old version of Etherpad no?

**Comment by JohnMcLear** [comment]
Confirmed on that pad instance, will try replicate on an up to date Etherpad.

**Comment by JohnMcLear** [comment]
``npm install ep_headings2 ep_table_of_contents``
exported .etherpad from framapad, imported to latest develop
clicking show table of contents provides error

```
Uncaught TypeError: $(...).context is undefined
    findTags http://192.168.77.180:9001/static/plugins/ep_table_of_contents/static/js/toc.js:37
    each http://192.168.77.180:9001/javascripts/lib/ep_etherpad-lite/static/js/ace2_common.js?callback=require.define&
...
Refreshing page
Clicking to Enable again
Still same error.


```

**Comment by JohnMcLear** [comment]
Fixed b7340f9..04843c1
